src/components/BannerBlock.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+
+interface BannerBlockProps {
+    content: {
+        text: string;
+        button: {
+            content: string;
+            link: string;
+        }
+    }
+}
+
+const BannerBlock = (props: BannerBlockProps) => (
+    <ApplicationDiv>
+      <FlexContainer>
+        <Text>{props.content.text}</Text>
+      </FlexContainer>
+      <FlexContainer>
+        <Button onClick={() => {}}>{props.content.button.content}</Button>
```

Comment:
if you can I'd say add in the link here. You just need to wrap this button in a <a> tag